src/containers/myTrees/test/selectors.test.ts

```diff
@@ -0,0 +1,174 @@
+import {
+  AsyncRequest,
+  AsyncRequestCompleted,
+  AsyncRequestNotStarted,
+  AsyncRequestFailed,
+} from '../../../utils/asyncRequest';
+import { AdoptedSites } from '../../treePage/ducks/types';
+import { SiteGeoData } from '../../../components/mapPageComponents/ducks/types';
+import { getMySites } from '../ducks/selectors';
+describe('My Trees Selectors', () => {
+  describe('getMySites', () => {
+    const dummySites: SiteGeoData = {
+      type: 'dumb',
+      name: 'fake sites',
+      features: [
+        {
+          type: 'dumb 0',
+          properties: {
+            id: 0,
+            treePresent: true,
+            lat: 100,
+            lng: 50,
+          },
+          geometry: {
+            type: 'fake 0',
+            coordinates: [0, 0],
+          },
+        },
+        {
+          type: 'dumb 1',
+          properties: {
+            id: 1,
+            treePresent: true,
+            lat: 50,
+            lng: 50,
+          },
+          geometry: {
+            type: 'fake 1',
+            coordinates: [1, 1],
+          },
+        },
+        {
+          type: 'dumb 2',
+          properties: {
+            id: 2,
+            treePresent: false,
+            lat: 100,
+            lng: 100,
+          },
+          geometry: {
+            type: 'fake 2',
+            coordinates: [2, 2],
+          },
+        },
+      ],
+    };
+
+    it('returns only sites that are adopted by the user', () => {
```

Comment:
📌  Not for this PR but one user was having a problem where their adopted sites weren't showing up. Fixed it by having them clear their local storage on Chrome but is there something we can do on our end? A bit weird since I thought we dispatch getAdoptedSites after adoption and when loading the my trees page

Reply:
iirc some people have had a similar issue with logging in when running the app locally. Not positive if this is something to do with this route specifically or if it's more widespread